presto-spi/src/main/java/com/facebook/presto/spi/constraints/TableConstraintsHolder.java

```diff
@@ -14,25 +14,39 @@
 package com.facebook.presto.spi.constraints;
 
 import com.facebook.presto.spi.ColumnHandle;
+import com.facebook.presto.spi.PrestoException;
 
 import java.util.ArrayList;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
+import static com.facebook.presto.spi.StandardErrorCode.NOT_SUPPORTED;
+import static java.lang.String.format;
 import static java.util.Collections.emptyList;
 import static java.util.Objects.requireNonNull;
 
 public class TableConstraintsHolder
 {
+    public static void validateTableConstraints(Collection<TableConstraint<String>> constraints)
```

Comment:
methods go after member variables.  See ordering in our codestyle: https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#code-style

Reply:
Fixed. The doc is not clear about static methods. I will update the code-style to make it more obvious